server/bundler/loader.js

```diff
@@ -177,5 +189,7 @@ module.exports = function( content ) {
 
 	this.addDependency( 'page' );
 
+	fillSectionCSSUrls( sections );
+
 	return getSectionsModule( sections );
```

Comment:
What about preventing `fillSectionCSSUrls()` from mutating data? Then we could do the following:

```javascript
return getSectionsModule( fillSectionCSSUrls( sections ) );
```

Reply:
I'll work on that 👍